### DISCUSSION TITLE
Server is returning empty

### DISCUSSION THREAD

**Discussion by GamalTawaf** [wevm/frog-archived#523] [Q&A]
Hello,
  I have a frog.fm server to generate images, after running for sometimes the images I get back are 0 in size 

  I'm running a bun server using pm2 
  The server is running but the return is empty.
The only way to solve it has been to restart the server.

any ideas on why this is happening? 

Thanks

**Comment by dalechyn** [comment]
Unfortunately without a min. repro or further explanation of the issue I won't be able to assist.
